portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,6 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-    <script src="map.js"></script>
   </head>
   <body onload="createMap();">
   <div id="content">
```

Comment:
should this be indented inside <body>?

Reply:
Yes, child elements should be indented.

Reply:
Has this been fixed? I.e. the indenting of all lines within the \<body> tags

Reply:
Now fixed all the indentation with an online html editor